### ISSUE TITLE
LAN Multiplayer support

### ISSUE THREAD

**Issue by Amoodee10** [FCL-Team/FoldCraftLauncher#659]
### Before making issue / 在发布提问之前...

- [X] I understand this is a request, not a wish.我明白这是在提出请求而不是在许愿。

### Description / 描述

LAN multiplayer would be a cool addition to this launcher

### Why / 原因

Bcz if HMCL PE can do it, why not if FCL can have it too

**Comment by 199825443** [comment]
hmcl pe的确有联机功能

**Comment by Amoodee10** [comment]
> Hmcl pe does have an online function

But hmcl is now broken u know. The creator wont touch it again

**Comment by 199825443** [comment]
> > Hmcl pe 确实具有在线功能
> 
> 但是 hmcl 现在已经坏了，你知道的。创作者不会再碰它

That's true, but if it's really developed, it's still good. You don't have to buy realmes or servers anymore

**Comment by 199825443** [comment]
如果有，我就不用模组或者服务器联机了

**Comment by 199825443** [comment]
> > Hmcl pe 确实具有在线功能
> 
> 但是 hmcl 现在已经坏了，你知道的。创作者不会再碰它

如果有，我就不用模组或者服务器联机了

**Comment by Amoodee10** [comment]
> > > Hmcl pe does have online functionality
> > 
> > 
> > But hmcl is broken now, you know. The creators won't touch it anymore
> 
> If so, I won't use mods or server connections anymore

U mean that HMCLPE IS STILL WORKING
